src/DGmethods/DGmodel_kernels.jl

```diff
@@ -20,6 +20,7 @@ const _ξ1x3, _ξ2x3, _ξ3x3 = Grids._ξ1x3, Grids._ξ2x3, Grids._ξ3x3
 const _M, _MI = Grids._M, Grids._MI
 const _x1, _x2, _x3 = Grids._x1, Grids._x2, Grids._x3
 const _JcV = Grids._JcV
+const _MH = Grids._MH
```

Comment:
```suggestion
```

Reply:
@jkozdon The main purpose of the PR is the have the slab metrics available and to test that they are correct. The main application we have for these currently is in the diagnostics, where I've had good results with using these metrics on a personal branch pending this gets merged. 

I'll checkout a fresh Manifest.